src/golang/lib/dynamic/engine.go

```diff
@@ -189,10 +192,22 @@ func CreateDynamicEngine(
 // If any step fails, it returns an error.
 func DeleteDynamicEngine(
 	ctx context.Context,
+	force bool,
```

Comment:
I feel we could be more specific on what 'force' means by using `skipSafetyCheck` or `skipActiveCheck`. We may just keep the `force` terminology in SDK and handlers that talks to SDK.

Reply:
I agree with this - I'd argue for something more specific though like "skipRunningPodsCheck". Otherwise, I'd argue for adding a function comment here.